core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java

```diff
@@ -144,7 +144,7 @@ public String getKafkaImage() {
 
     @Deprecated
     public String getOracleImage() {
-        return getEnvVarOrUserProperty("oracle.container.image", null);
+        return getEnvVarOrProperty("oracle.container.image", null);
```

Comment:
@rnorth shouldn't it be `getImage`?

Reply:
Not for this one... `getImage` provides a substitute name (or the same name) for a given image.

For Oracle we have no default image name at all, so `getImage` has nothing to substitute. I don't like this, but it's one of those things with the Oracle module. 

As a longer term way to tidy this up, I hope that we can work towards removing 'zero-args' instantiation of Oracle and all other containers: so that both for the constructor in code, and in JDBC URLs, an image name must be specified.